src/design-tokens/tier-1-definitions/typography-presets.css

```diff
@@ -106,10 +106,9 @@
 @define-mixin eds-typography-preset-8 $important {
   font-family: var(--eds-font-family-tertiary) $important;
   font-weight: var(--eds-font-weight-regular) $important;
-  font-size: var(--eds-font-size-md) $important;
-  line-height: var(--eds-line-height-sm) $important;
+  font-size: var(--eds-font-size-xs) $important;
+  line-height: var(--eds-size-1-and-half) $important;
```

Comment:
I didn't add tokens for the new line heights because, honestly, I'm lazy and I don't want to rabbit hole into type sizes right now.

Reply:
that's fine lol, could you leave comments though that we should update with line-height tokens when they're ready